tests/RequestParserTest.php

```diff
@@ -252,7 +252,7 @@ private function createPostWithContent()
         $data .= "Host: localhost:8080\r\n";
         $data .= "User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.7; rv:32.0) Gecko/20100101 Firefox/32.0\r\n";
         $data .= "Connection: close\r\n";
-        $data .= "Content-Type: application/x-www-form-urlencoded\r\n";
+        $data .= "Content-Type: application/x-www-form-urlencoded; charset=utf-8\r\n";
```

Comment:
@caseyamcl No worries we all have a life outside github; Yes that has my preference. That was we can easily include more test cases by simple adding more entries in the data provider :+1: